packages/nimble-components/src/table/tests/table.stories.ts

```diff
@@ -110,7 +110,9 @@ const metadata: Meta<TableArgs> = {
             id-field-name="${x => dataSetIdFieldNames[x.data]}"
             data-unused="${x => x.tableRef.setData(dataSets[x.data])}"
         >
-            <nimble-table-column-text field-name="firstName" placeholder="no value">First Name</nimble-table-column-text>
+            <nimble-table-column-text field-name="firstName" placeholder="no value">
```

Comment:
Let's add a non-text test case to the matrix story too. Will be good to verify that configuration of other properties like theming continues to work.